spec/unit/puppet/type/keycloak_realm_spec.rb

```diff
@@ -49,6 +49,7 @@
     events_listeners: ['jboss-logging'],
     admin_events_enabled: :false,
     admin_events_details_enabled: :false,
+    offline_session_max_lifespan_enabled: false,
```

Comment:
```suggestion
    offline_session_max_lifespan_enabled: :false,
```

Reply:
The unit tests will skip checking for default if value is `false` and technically the default is a symbol.

https://github.com/treydock/puppet-module-keycloak/blob/2e099ed88f6ec9156b192905813016d3e0890409/spec/unit/puppet/type/keycloak_realm_spec.rb#L155